Gemfile

```diff
@@ -57,12 +52,17 @@ group :test do
   gem 'oga'
 
   # faster json libraries
-  if !ENV['PURE_RUBY']
+  unless ENV['PURE_RUBY']
     gem 'json', '1.8.3' if RUBY_VERSION == '1.9.3'
     if ENV['OLD_OJ']
       gem 'oj', '1.3.0'
     else
-      gem 'oj' unless RUBY_VERSION == '1.9.3'
+      if RUBY_VERSION == '1.9.3'
+        # oj drop support for Ruby under 2.0 since 3.3.5
+        gem 'oj', '<= 3.3.4'
+      else
+        gem 'oj'
+      end
     end
```

Comment:
These are grouped under :test dependencies.. but these should be labeled as core shouldn't they?